eng/pipelines/templates/jobs/archetype-sdk-integration.yml

```diff
@@ -92,18 +118,45 @@ jobs:
       - script: |
           node common/scripts/install-run-rush.js install
         displayName: "Install dependencies"
+        condition: and(succeeded(),ne(variables['DependencyVersion'],''))
 
       # Option "-p max" ensures parallelism is set to the number of cores on all platforms, which improves build times.
       # The default on Windows is "cores - 1" (microsoft/rushstack#436).
       - script: |
           node common/scripts/install-run-rush.js build -t "${{parameters.PackageName}}" --verbose -p max
         displayName: "Build libraries"
+        condition: and(succeeded(),ne(variables['DependencyVersion'],''))
 
       # Option "-p max" ensures parallelism is set to the number of cores on all platforms, which improves build times.
       # The default on Windows is "cores - 1" (microsoft/rushstack#436).
       - script: |
           node common/scripts/install-run-rush.js build:test -t "${{parameters.PackageName}}" --verbose -p max
         displayName: "Build test assets"
+        condition: and(succeeded(),eq(variables['DependencyVersion'],''))
+
+      - task: DeleteFiles@1
```

Comment:
Why are we deleting here? How does this relate to the rush unlink we are doing later?